extension/gulpfile.js

```diff
@@ -88,8 +88,11 @@ gulp.task('browserify', () => {
   ], {read: false})
     .pipe(tap(file => {
       file.contents = browserify(file.path, {
+        fullPaths: true,
```

Comment:
adds ~94KB to lighthouse-background.js, plus the local paths of whoever is building the extension. Running the extension, it appears to work without `fullPaths`, so, remove?